src/main/java/ml/shifu/shifu/core/Scorer.java

```diff
@@ -70,10 +70,11 @@
      */
     private Map<Integer, Map<String, Integer>> binCategoryMap = new HashMap<Integer, Map<String, Integer>>();
 
+
     /**
-     * For neural network, if output the hidden neurons
+     * For neural network, if output the first
      */
-    private int outputHiddenLayerIndex = 0;
+    private boolean outputFirstHiddenLayer = false;
```

Comment:
outputHiddenLayerIndex  as a int is new version, please make sure you merge to latest shifu code